meta/signature/draft-1.json

```diff
@@ -274,7 +290,32 @@
         },
         "Perturbation": {
           "description": "The perturbation performed in the study", 
-          "type": "string"
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "Name": {
+                "description": "Name of the Perturbation",
+                "type": "string"
+              },
+              "Original String": {
+                "description": "The original string",
+                "type": "string"
+              }
+            },
+            "oneOf": [
+              {
+                "required": [
+                  "Original String"
+                ]
+              },
+              {
+                "required": [
+                  "Name"
+                ]
+              }
+            ]
```

Comment:
What makes the Name different from Original String?

Reply:
Hmm... original string is the original wording of the metadata on the gmt files (i.e. not checked against an ontology, that's the name). I retained the original string as it contains some other information that might be useful + for validation later. Then again, we are already storing the original gmt metadata string so it is also possible to remove this altogether.

Reply:
6d08aacc5326b0394a565bad2f6ec59b39316f2a